frontend/src/Components/RoleSelection.js

```diff
@@ -92,11 +93,11 @@ const RoleSelection = ({ question, matchedWith }) => {
       };
       const response = await axios.request(config);
       console.log(response.data);
-      //  if (title === "Mentor") {
-      navigate("/mentor-flow-4");
-      //  } else if (role === "Mentee") {
-      //    navigate("/mentee-flow-1");
-      //  }
+       if (title === "Mentor") {
+      navigate("/mentor-flow-2");
+       } else if (title === "Mentee") {
+         navigate("/mentee-flow-2");
+       }
```

Comment:
we can just do else block like:
```suggestion
       } else {
         navigate("/mentee-flow-2");
       }
```